libvast/vast/fwd.hpp

```diff
@@ -70,14 +71,11 @@ struct vector_type;
 
 // -- smart pointers -----------------------------------------------------------
 
+using chunk_ptr = caf::intrusive_ptr<chunk>;
```

Comment:
If we're going to drop all the vertical separation: should we also sort this block like we do with classes?

Reply:
Yeah, I'll change that.